test/DynamoCoreUITests/RecordedTests.cs

```diff
@@ -1058,9 +1058,11 @@ public void TestUpdateNodeCaptions_DS()
             Assert.AreEqual("CBN", cbn.NickName);
         }
 
-        [Test, RequiresSTA, Category("Failure")]
+        [Test, RequiresSTA]
         public void ReExecuteASTTest()
         {
+            DynamoUtilities.DynamoPathManager.Instance.AddPreloadLibrary("FFITarget.dll");
```

Comment:
We might need an API to remove certain library from preload sequence, when library loader is re-initialized.